modules/mobile/app/controllers/mobile/v0/locations_controller.rb

```diff
@@ -4,18 +4,22 @@ module Mobile
   module V0
     class LocationsController < ApplicationController
       def show
+        # not helpful
         Rails.logger.info('Mobile Location', param_id: params[:id])
```

Comment:
Skip all of these for now because I'm so perplexed by why I don't see any logging from the service this uses.